hard-protocol-home.js

```diff
@@ -320,12 +350,17 @@ const updateDisplayValues = async(denoms) => {
   const hardSupplyRewardsPerYearByDenom = await getRewardPerYearByDenom(siteData);
   siteData['hardSupplyRewardsPerYearByDenom'] = hardSupplyRewardsPerYearByDenom;
 
+  const supplyApyByDenom = await getSupplyApyPerYearByDenom(siteData);
+  siteData['supplyApyByDenom'] = supplyApyByDenom;
```

Comment:
With the comment above, you can remove these two lines of code as you will have the structured data already in `siteData['interestRates']`